cli/man/transact-workload.1.md

```diff
@@ -85,7 +90,8 @@ transact workload \
   --update 5 \
   --smallbank-seed 10 \
   --smallbank-num-accounts 5 \
-  --targets "http://0.0.0.0:8089/scabbard/jEWSK-jdjSM/a001;http://0.0.0.0:8088/scabbard/jEWSK-jdjSM/
+  --targets "http://0.0.0.0:8089/scabbard/jEWSK-jdjSM/a001;http://0.0.0.0:8088/scabbard/jEWSK-jdjSM/ \
+  --duration 24
```

Comment:
should this be 24h ?

Reply:
Forgot to update this after changing the value that duration takes, fixed now